CONTRIBUTING.rst

```diff
@@ -83,6 +85,7 @@ Development Workflow
    * Finally, we recommend you install pre-commit which checks
      that your code matches formatting guidelines::
 
+       pip install pre-commit
```

Comment:
`pre-commit` is there in `requirements/developer.txt`([see here](https://github.com/networkx/networkx/blob/main/requirements/developer.txt#L4)). And with https://github.com/conda-forge/staged-recipes/pull/28089 there won't be a need to explicitly install `changelist` also. 

Thank you for reporting this!